bitnami/harbor/templates/core/core-svc.yaml

```
@@ -19,5 +19,10 @@ spec:
     - name: {{ ternary "https" "http" .Values.internalTLS.enabled }}
       port: {{ ternary .Values.core.service.ports.https .Values.core.service.ports.http .Values.internalTLS.enabled }}
       targetPort: {{ ternary "https" "http" .Values.internalTLS.enabled }}
+    {{- if .Values.metrics.enabled }}
+    - name: {{ template "harbor.metricsPortName" . }}
+      port: {{ .Values.core.service.ports.metrics }}
+      targetPort: {{ template "harbor.metricsPortName" . }}
```

Again.. I don't see any value on using a helper that hardcodes the name. Also, add in the name the protocol as prefix:
```suggestion
    - name: http-metrics
      port: {{ .Values.core.service.ports.metrics }}
      targetPort: metrics
```